docs/compass/03-packages-api.md

```diff
@@ -0,0 +1,81 @@
+# Packages API
+
+## Introduction
+
+On Kyma Runtime, every Application is represented as a single Service Class, and every Package of a given Application is represented as a single Service Plan.
```

Comment:
I would elaborate a little bit more about packages itself, for example on what is the added value of that feature.

Reply:
agree, I think that this sentence should be first:
> This document describes API for Packages, which groups multiple API Definitions, Event Definitions and Documents.